ugvc/pipelines/mrd/sorter_stats_to_mean_coverage.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+import argparse
+
+import numpy as np
+
+from ugvc.utils.metrics_utils import read_effective_coverage_from_sorter_json
+
+
+def parse_args(argv: list[str]) -> argparse.Namespace:
+    parser = argparse.ArgumentParser(prog="annotate featuremap", description=run.__doc__)
+    parser.add_argument(
+        "-i",
+        "--sorter-stats-json",
+        type=str,
+        required=True,
+        help="input sorter stats JSON file",
+    )
+    parser.add_argument(
+        "-o",
+        "--output-file",
+        type=str,
+        required=True,
+        help="Path of output file",
+    )
+    return parser.parse_args(argv[1:])
+
+
+def sorter_stats_to_mean_coverage(sorter_stats_json: str, output_file: str):
+    """
+    Read effective coverage metrics from sorter JSON file and write the up-rounded result to a file.
+
+    Parameters
+    ----------
+    sorter_stats_json : str
+        Path to Sorter statistics JSON file.
+    output_file : str
+        Path to output file.
+
+    """
+    mean_cvg = np.ceil(read_effective_coverage_from_sorter_json(sorter_stats_json)[0])
```

Comment:
Could you give some clarification what effective_coverage means? Either here or in `read_effective_coverage_from_sorter_json`

Reply:
In this case it's just the mean coverage, fixed, and clarified in the other function